lib/MusicBrainz/Server/Entity/LinkType.pm

```diff
@@ -123,7 +124,18 @@ around TO_JSON => sub {
     my ($orig, $self) = @_;
 
     my $json = $self->$orig;
+
+    $json->{cardinality0} = $self->entity0_cardinality;
+    $json->{cardinality1} = $self->entity1_cardinality;
+    $json->{deprecated} = boolean_to_json($self->is_deprecated);
+    $json->{hasDates} = boolean_to_json($self->has_dates);
+    $json->{id} = $self->id;
     $json->{link_phrase} = $self->link_phrase;
+    $json->{long_link_phrase} = $self->long_link_phrase;
+    $json->{orderableDirection} = $self->orderable_direction;
```

Comment:
Updated those too. :) Nothing requires them to match the Moose attribute names specifically, at least at present, but I prefer to use the Moose ones since they're already prevalent throughout the codebase. Having consistent property names just avoids unnecessary friction and confusion. For context, we'd been discussing having a standardized data format since at least 2014, to help with edit diffs and the like. It'd even have the nice property that we'd be able to pass a piece of entity JSON (in standard format) to a Moose entity constructor in the backend, and have it just work without having to remap property names.